ovirtlib4/vms.py

```diff
@@ -121,6 +138,17 @@ class VmNic(CollectionEntity):
     def __init__(self, *args, **kwargs):
         super(). __init__(*args, **kwargs)
 
+    def get_profile(self):
```

Comment:
I would be more explicit about this being a vnicprofile rather then just a profile